draft-ietf-rats-eat.md

```diff
@@ -1182,6 +1196,12 @@ The profile document should go into the full detail.
 Similar to, or perhaps the same as Verification Key Identification, the profile may wish to specify how Endorsements are to be identified.
 However note that Endorsement Identification is optional, where as key identification is not.
 
+### Freshness
+
+Just about every use case will require some means of knowing the EAT is recent enough and not a replay of an old token.
+The profile should describe how freshness is achieved.
+The section on Freshness in {{RATS-Architecture}} describes several solutions to achieve this.
```

Comment:
Maybe we could make it clearer here that one is not bound to the freshness models examined in {{RATS-Architecture}}.  E.g., a protocol embedding EAT could use a time notion based on some kind of event stored in Merkle tree, or anything else really as long as it fits the bill.